src/backend/mailbox_manager/models.py

```diff
@@ -202,6 +218,9 @@ class Mailbox(BaseModel):
         choices=MailboxStatusChoices.choices,
         default=MailboxStatusChoices.PENDING,
     )
+    dn_email = models.EmailField(_("email"), blank=True, unique=True, editable=False)
```

Comment:
what is dn_ ? isn't it a potential issue to store both this and the local_part ?

Reply:
"dn_" stands for "denormalized". In the first place, I tried to add a generated field but it does not work with "join" (it was a concat of "local_part", "@" and "domain__name" :boom: ), so I used a field which is computed on save to store the denormalized email value. 

For sure, if you make `Mailbox.objects.filter().update(local_part="blah")`, the `dn_email` will be broken... 

The `dn_email` is required to make the Django admin work for mailbox with password...

I can add a comment to say we should only use this field for the Django admin.

Reply:
Could this be a reason to do the reverse and compute local_part while only storing the full email?

Reply:
We could, but it implies a bit of work, if we need it we may do it in another PR if it's ok for you.